features/fixtures/mazerunner/cxx-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/CXXComplexMetadataScenario.kt

```diff
@@ -0,0 +1,55 @@
+package com.bugsnag.android.mazerunner.scenarios
+
+import android.content.Context
+import com.bugsnag.android.Bugsnag
+import com.bugsnag.android.Configuration
+import com.bugsnag.android.mazerunner.log
+import java.util.Date
+
+/**
+ * Sends a native crash to Bugsnag which has complicated metadata values.
+ */
+internal class CXXComplexMetadataScenario(
+    config: Configuration,
+    context: Context,
+    eventMetadata: String?
+) : Scenario(config, context, eventMetadata) {
+
+    companion object {
+        init {
+            System.loadLibrary("cxx-scenarios")
+        }
+    }
+
+    private external fun crash()
+
+    override fun startScenario() {
+        super.startScenario()
+        val longStr = "Have you tried turning it off and on again and off and on again and off" +
+            " and on again and off and on again and off and on again and off and on again" +
+            " and on again and off and on again and off and on again and off and on again?"
+        val map = mapOf(
+            "int_array" to intArrayOf(-5, 1000, 29),
```

Comment:
```suggestion
            "int_array" to intArrayOf(-5, 1000, 29, Int.MAX_VALUE, Int.MIN_VALUE),
```

Assuming it doesn't do anything weird with Mazerunner?